Sources/MapboxNavigation/NavigationViewportDataSource.swift

```diff
@@ -53,31 +59,17 @@ public class NavigationViewportDataSource: ViewportDataSource {
      */
     public var defaultAltitude: CLLocationDistance = 1000.0
```

Comment:
Is it used anywhere?

Reply:
Yep, it's needed when active guidance navigation just starts to prevent huge jump from initial location to current one.